StortSpelProjekt/Game/src/Gamefiles/Components/PlayerInputComponent.cpp

```diff
@@ -48,9 +48,6 @@ component::PlayerInputComponent::PlayerInputComponent(Entity* parent, unsigned i
 	m_UpdateShootId = -1;
 	m_UpdateDashId = -1;
 
-	m_DegreesToTurnTo = 0;
-	m_RadiansToTurn = 0;
-
 	m_TurningTimer = 0.0f;
 	m_TurningInterval = 0.0f;
```

Comment:
Looks like this should also get removed? Can't find any logic with these variables, they are only set to stuff.

Reply:
There is a logic to them, you just don't understand it. They have less than nothing to do with this task